doc/cookbook/source/examples/classifier/hierarchical.rst

```diff
@@ -0,0 +1,36 @@
+=======================
+Hierarchical Clustering
```

Comment:
Currently all clustering methods are under classifier entry..but it should be hierarchical